packages/inter-protocol/src/proposals/add-auction.js

```diff
@@ -166,12 +168,21 @@ export const addAuction = async (
     }),
   );
 
+  auctionInstance.reset();
+  await auctionInstance.resolve(governedInstance);
+  // belt and suspenders; the above is supposed to also do this
+  await E(E(agoricNamesAdmin).lookupAdmin('instance')).update(
```

Comment:
`agoricNamesAdmin` is a big hammer; it's unfortunate that `resolve()` alone didn't do the trick.

here's hoping for some later convenient time to figure out why not

Reply:
the comment reads "belt-and-suspenders" as if the belt sometimes works. Does it not?